mod/groups/views/default/groups/invitationrequests.php

```diff
@@ -46,6 +63,14 @@
 		}
 	}
 	echo '</ul>';
+
+	if (!empty($count)) {
+		echo elgg_view('navigation/pagination', array(
```

Comment:
So icky to have to manage this ourselves. This makes me like the idea of using elgg_list_entities, bat passing in a closure to render items...